src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] It would be great if you remove all the boilerplate files other than `index.html` from the public folder to follow the project requirements :+1:
- [ ] Also, it would be great if you remove all the boilerplate files (*texts*, *images*, and *styles*) from the src folder to make your project follow the requirements :blush: